package.json

```diff
@@ -13,10 +13,10 @@
   "devDependencies": {
     "@babel/cli": "^7.10.4",
     "@babel/core": "^7.11.0",
-    "@babel/eslint-config-internal": "link:./eslint/babel-eslint-config-internal",
-    "@babel/eslint-parser": "^7.11.0",
-    "@babel/eslint-plugin-development": "^7.11.0",
-    "@babel/eslint-plugin-development-internal": "link:./eslint/babel-eslint-plugin-development-internal",
+    "@babel/eslint-config-internal": "portal:./eslint/babel-eslint-config-internal",
+    "@babel/eslint-parser": "portal:./eslint/babel-eslint-parser",
```

Comment:
Thanks to the `portal:` protocol available only in Yarn 2, Babel can be linted by latest `@babel/eslint-parser`. cc @kaicataldo

Reply:
You could use the workspace protocol instead, which does a better job here

Reply:
This is the root `package.json`, which is not part of any workspace. Use `workspace` protocol here will throw
```
Workspace not found
```

Reply:
The root is technically a workspace, but anyways, it should work as the eslint folders are declared as a workspaces. Jest does this as well https://github.com/facebook/jest/blob/c9c8dba4dd8de34269bdb971173659399bcbfd55/package.json#L56, If you tried before https://github.com/babel/babel/pull/11962/commits/4e94e2a56eb96bf5566eb829a2bf23a02cd2df0a it makes sense that it failed

Reply:
I tried after 3d1e4c3 and it failed. I guess it is related to `enableTransparentWorkspaces: false` in `yarnrc`.

Reply:
Hm, just tested with master and got the same result, however changing it to this works:
```diff
diff --git a/package.json b/package.json
index be3015833..11d9a134b 100644
--- a/package.json
+++ b/package.json
@@ -13,10 +13,10 @@
   "devDependencies": {
     "@babel/cli": "^7.10.4",
     "@babel/core": "^7.11.0",
-    "@babel/eslint-config-internal": "portal:./eslint/babel-eslint-config-internal",
-    "@babel/eslint-parser": "portal:./eslint/babel-eslint-parser",
-    "@babel/eslint-plugin-development": "portal:./eslint/babel-eslint-plugin-development",
-    "@babel/eslint-plugin-development-internal": "portal:./eslint/babel-eslint-plugin-development-internal",
+    "@babel/eslint-config-internal": "workspace:*",
+    "@babel/eslint-parser": "workspace:*",
+    "@babel/eslint-plugin-development": "workspace:*",
+    "@babel/eslint-plugin-development-internal": "workspace:*",
     "@babel/plugin-proposal-dynamic-import": "^7.10.4",
     "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
     "@babel/plugin-transform-flow-strip-types": "^7.10.4",
```

Or if you want to be explicit:
```diff
diff --git a/package.json b/package.json
index be3015833..b31abfce9 100644
--- a/package.json
+++ b/package.json
@@ -13,10 +13,10 @@
   "devDependencies": {
     "@babel/cli": "^7.10.4",
     "@babel/core": "^7.11.0",
-    "@babel/eslint-config-internal": "portal:./eslint/babel-eslint-config-internal",
-    "@babel/eslint-parser": "portal:./eslint/babel-eslint-parser",
-    "@babel/eslint-plugin-development": "portal:./eslint/babel-eslint-plugin-development",
-    "@babel/eslint-plugin-development-internal": "portal:./eslint/babel-eslint-plugin-development-internal",
+    "@babel/eslint-config-internal": "workspace:eslint/babel-eslint-config-internal",
+    "@babel/eslint-parser": "workspace:eslint/babel-eslint-parser",
+    "@babel/eslint-plugin-development": "workspace:eslint/babel-eslint-plugin-development",
+    "@babel/eslint-plugin-development-internal": "workspace:eslint/babel-eslint-plugin-development-internal",
     "@babel/plugin-proposal-dynamic-import": "^7.10.4",
     "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
     "@babel/plugin-transform-flow-strip-types": "^7.10.4",

```

Reply:
@merceyz Thanks! I go with `workspace:*` for simplicity. It seems like a bug to me if `"workspace:eslint/babel-eslint-parser"` is working while `"workspace:./eslint/babel-eslint-parser"` is bailing. Should I file an issue for that?

Reply:
Yes please :+1:

Reply:
Filed https://github.com/yarnpkg/berry/issues/1712.